lib/schedule/utils.js

```diff
@@ -69,7 +77,9 @@ const pull = revision => {
  *
  * @return Promise
  */
-const push = revision => {
+const push = (revision, apps) => {
+  let options;
```

Comment:
This can be a `const` with the `updated` options, then line 106 you can change the trigger just as-is. `const` (and our linting) doesn't about reassigning parameters, only the ident itself